src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
It seems that the the code path that executes ``FunctionLength = *(PTR_ULONG)(ImageBase + FunctionLength) & 0x3ffff;`` in ARM implementation seems to corresponds to the current x86 implementation.